tests/autoyast/installation.pm

```diff
@@ -206,6 +211,17 @@ sub run {
             push(@needles, 'autoyast-license') if (get_var('AUTOYAST_LICENSE'));
             next;
         }
+        elsif (match_has_tag('untrusted-ca-cert')) {
+            send_key 'alt-t';
+            wait_still_screen 5;
+            next;
+        }
+        elsif (match_has_tag('ERROR-removing-package')) {
+            send_key 'alt-i';    # ignore
```

Comment:
Do we have a bug for this scenario? I would add soft_failure here as it doesn't look like expected behavior.

Reply:
Yes, we have a bug bsc#1084997: removing packages libburnia-tools failed during upgrade.

ERROR-removing-package could happen on any package, it would be misunderstanding if I add record_soft_failure("bsc#1084997") here but another package failed to remove after libburnia-tools issue fixed.

So I prefer to create package specific needle with "workaround" label, just like the existing one for libburnia-tools:
ERROR-removing-package-bsc1084997-20180320.json
ERROR-removing-package-bsc1084997-20180320.png

Reply:
@rwx788 about your concern I had already talked offline with @QingmingSu. So let me merge it since we need autoupgrade on o.s.d. asap.

Reply:
Sure, let's proceed then.